### PR TITLE
Moves HorizontalContentSeriesFeedConnected to the ui-connected package

### PR DESCRIPTION
**Pull request by burkeshartsis**
### What does this PR do, or why is it needed?
This PR moves the formerly named `HorizontalContentFeed` to the `ui-connected` package and renames it to `HorizontalContentSeriesFeedConnected`. It also does some mild clean up and refactoring to make it follow already established paradigms and to enabled placeholder loading states. Please review this closely and test on both platforms. 

### What design trade-offs/decisions were made?
N/A
### How do I test this PR?
open app and tap on a series or series child. The feed at the bottom should show up and display a loading state. 

### What automated tests did you write?
No change in test coverage.

## TODO:

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS
- [x] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-ui-connected/src/HorizontalContentSeriesFeedConnected/HorizontalContentSeriesFeedConnected.js [hunk 1]
```diff
@@ -8,35 +8,57 @@ import { HorizontalTileFeed, TouchableScale } from '@apollosproject/ui-kit';
...
     const itemId = get(item, 'id', '');
     const disabled = get(item, 'id', '') === this.props.contentId;
+    const isLoading = get(item.node, 'isLoading');
+    const labelText = get(item.node, 'parentChannel.name', null);
+
     return (
       <TouchableScale
         onPress={() => this.handleOnPressItem(item)}
-        disabled={disabled}
+        disabled={isLoading || disabled}
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/HorizontalContentSeriesFeedConnected/HorizontalContentSeriesFeedConnected.js]: 🔥

File: packages/apollos-ui-connected/src/HorizontalContentSeriesFeedConnected/HorizontalContentSeriesFeedConnected.js [hunk 2]
```diff
@@ -8,35 +8,57 @@ import { HorizontalTileFeed, TouchableScale } from '@apollosproject/ui-kit';
...
       >
         <HorizontalContentCardConnected
+          labelText={labelText}
           contentId={itemId}
           disabled={disabled}
+          isLoading={isLoading}
+          // If we are loading we need to assume a typename so HorizontalContentCardConnected knows what "type" to render
+          __typename={isLoading ? 'MediaContentItem' : get(item, '__typename')}
```
[the remark this answers is not in the document]
Comment (burkeshartsis) [reply packages/apollos-ui-connected/src/HorizontalContentSeriesFeedConnected/HorizontalContentSeriesFeedConnected.js]: Great catch 🤦‍♂ Fixed in 42618df

File: packages/apollos-ui-connected/src/HorizontalContentSeriesFeedConnected/HorizontalContentSeriesFeedConnected.js [hunk 3]
```diff
@@ -8,35 +8,58 @@ import { HorizontalTileFeed, TouchableScale } from '@apollosproject/ui-kit';
...
         onPress={() => this.handleOnPressItem(item)}
-        disabled={disabled}
+        disabled={isLoading || disabled}
       >
         <HorizontalContentCardConnected
+          labelText={labelText}
           contentId={itemId}
           disabled={disabled}
+          isLoading={isLoading}
+          __typename={get(item, '__typename')}
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/HorizontalContentSeriesFeedConnected/HorizontalContentSeriesFeedConnected.js]: This is such a nit pick, but should we move this to the list of other `const`'s under line 50? Or do the inverse and get rid of all the `const`'s and move all the `get`s inline?

Comment (burkeshartsis) [reply packages/apollos-ui-connected/src/HorizontalContentSeriesFeedConnected/HorizontalContentSeriesFeedConnected.js]: What if I did it in reverse and removed the unneeded variables and moved lines 47 and 50 inline?

Comment (vinnyjth) [reply packages/apollos-ui-connected/src/HorizontalContentSeriesFeedConnected/HorizontalContentSeriesFeedConnected.js]: That's my second suggestion, I like that idea.

Comment (burkeshartsis) [reply packages/apollos-ui-connected/src/HorizontalContentSeriesFeedConnected/HorizontalContentSeriesFeedConnected.js]: 👍 Addressed in d07c94d

**Comment by vinnyjth** [comment]
Let's merge this as soon as the tests pass / we make them pass.

**Pull request merged**